### ISSUE TITLE
Automated deployment

### ISSUE THREAD

**Issue by kaihendry** [unee-t/frontend#36]
Via https://github.com/unee-t/frontend/blob/master/.travis.yml & [travis script deployment](https://docs.travis-ci.com/user/deployment/script/) master branch commits will be automatically deployed via ecs-cli to the 812644853088 ECS cluster into the meteor service.

AKA https://dev.case.unee-t.com/

**Comment by kaihendry** [comment]
Btw to do a re-deploy on ECS, restarting the build might suffice: https://travis-ci.org/unee-t/frontend

ecs-cli is a bit silly in the sense that if the docker compose file didn't change, it won't create a new task.

**Comment by kaihendry** [comment]
Not sure why I stick to ecs-cli when it frustratingly can't do some basic things.

* https://github.com/aws/amazon-ecs-cli/issues/402
* https://github.com/aws/amazon-ecs-cli/issues/403

**Comment by kaihendry** [comment]
Blocked by https://github.com/unee-t/frontend/issues/55

**Comment by kaihendry** [comment]
Ok, this is now working on both:

* [meteor](https://travis-ci.org/unee-t/frontend/builds/326239428?utm_source=github_status&utm_medium=notification)
* [bugzilla](https://travis-ci.org/unee-t/bugzilla-customisation)
